storage/leveldb/leveldb.go

```diff
@@ -20,6 +21,8 @@ var _ storage.Persister = (*DB)(nil)
 
 // read + write + execute for owner only
 const rwxOwner = 0700
+const mkdirAllFunction = "MkdirAll"
```

Comment:
```suggestion
const mkdirAllFunction = "mkdirAll"
```
to start with lowercase, the same as openLevelDBFunction

Reply:
done